custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -389,10 +395,16 @@ def pre_aggregation_queries(self):
 
     def create_temporary_table(self):
         return """
-        CREATE TABLE \"{table}\" (LIKE child_health_monthly) PARTITION BY LIST (state_id);
+        CREATE UNLOGGED TABLE \"{table}\" (LIKE child_health_monthly) PARTITION BY LIST (state_id);
```

Comment:
Does this Unlogged add any performance boost?

Reply:
Yeah, huge one because it cuts down on disk i/o by not having to write the WAL files.